cloud/services/compute/instances/reconcile_test.go

```diff
@@ -228,4 +335,5 @@ func TestService_createOrGetInstance(t *testing.T) {
 			}
 		})
 	}
+
```

Comment:
remove this new line here, lint is not happy :)

Reply:
Thanks :)